xml/System.Numerics/Vector`1.xml

```diff
@@ -92,7 +92,18 @@
       <Docs>
         <param name="values">A read-only span of bytes that contains the values to add to the vector. The span must contain at least <see cref="P:System.Numerics.Vector`1.Count" /> elements.</param>
         <summary>Constructs a vector from the given read-only span of bytes.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+ The type `T` can be any primitive numeric type (that is, a numeric type whose <xref:System.Type.IsPrimitive%2A?displayProperty=nameWithType> property returns `true`.  
```

Comment:
"primitive numeric type" is a confusing term due to differences in what C# and the runtime consider to be "primitive" and what some languages consider to be "numeric".

It might be better to be explicit and list out the following 10 types somewhere: `Byte`, `Double`, `Int16`, `Int32`, `Int64`, `SByte`, `Single`, `UInt16`, `UInt32`, `UInt64`.

This might also expand in the future to include types that neither the language nor the runtime consider "primitive" (such as `Half` or `BFloat16`).

Reply:
Fixed; I pulled them out to the top of the document and listed out all 10 types.